spec

```diff
@@ -1514,7 +1514,6 @@ endseg
 beginseg
     name "ovl_Demo_Effect"
     include "build/src/overlays/actors/ovl_Demo_Effect/z_demo_effect.o"
-    include "build/data/overlays/actors/z_demo_effect.data.o"
     include "build/data/overlays/actors/z_demo_effect.reloc.o"
```

Comment:
You'll want to update the reloc entry as well. See other matched actors for an example